src/util/string.h

```diff
@@ -173,4 +222,12 @@ template <typename T1, size_t PREFIX_LEN>
 }
 } // namespace util
 
+namespace tinyformat {
+template <typename... Args>
+std::string format(util::ConstevalFormatString<sizeof...(Args)> fmt, const Args&... args)
```

Comment:
can we still have errors like [this](https://github.com/bitcoin/bitcoin/blob/master/src/logging.h#L253-L265) after this change?